core/capability-negotiation-3.2.md

```diff
@@ -42,7 +42,8 @@ If no other arguments are present, version 3.1 is assumed.
 ### Capabilities with values
 
 Servers MAY specify additional data for each capability using the
-`<name>[=<value>]` format.
+`<name>[=<value>]` format for `CAP LS` and `CAP NEW`.  `CAP ACK` and
+`CAP NAK` replies MUST NOT contain any values.
```

Comment:
Alright, that makes sense.  I was concerned some servers might simply do `<name>=<value>` in all responses, but that may be an unfounded worry.

```md
 Servers MAY specify additional data for each capability using the
`<name>[=<value>]` format for `CAP LS` and `CAP NEW`.
```

Then, for the Errata, keep a note about the situation as it is right now.  This should more clearly inform to anyone whom originally implemented values in all replies:
```md
## Errata

A previous version of this specification did not specify when to use
the `<name>[=<value>]` format.  This was clarified to limit capability
values to `CAP LS` and `CAP NEW` replies.

Servers currently including values in `CAP ACK` and `CAP NAK` replies
MUST modify these replies to not include values.
```

Alternatively, should the Errata avoid mentioning them as well?